language/move-vm/types/src/logger.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
why are we introducing a new logging framework :O?

cc: @bmwill

Reply:
LOL, that is not quite accurate as mentioned offline as well, also I have added you to the right PR #6237 sorry I did not do it sooner. This is just a test to verify that, in fact, the structure logging that is in common/logger is in fact working as expected

Reply:
also in this specific PR please take a look at `language/libra-vm/src/logger.rs` to get an answer to your question (or your concerns)